egs/gale_arabic/s5d/local/process_xml.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
```

Comment:
please write 'python3' explicitly in both scripts (i.e. `#!/usr/bin/env python3`)

Reply:
Oh and please make sure it actually does run under py3 if youused py2.7

Reply:
OK, it runs under python3, I will write python3.